composer.json

```diff
@@ -20,9 +20,10 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
+        "ext-simplexml": "*",
         "behat/transliterator": "^1.2",
-        "jms/metadata": "^1.6",
+        "jms/metadata": "^2.0",
```

Comment:
The changelog for https://github.com/dustin10/VichUploaderBundle/releases/tag/1.9.0 says "allow for `jms/metadata` version 2", but that's not *allow for*, that's *require*! As a result, I can't install `vich/uploader-bundle` ^1.13 (needed for `league/flysystem-bundle` support) in a project that has a transitive dependency on `jms/metadata` 1.7 😢

I don't understand https://github.com/dustin10/VichUploaderBundle/pull/918#issuecomment-464262678

> 2. I see you did `"jms/metadata": "^1.6|^2.0"` for the compatibility with php 7.1 I guess ?
>    It won't work because `::loadMetadataFromFile()` on jms/metadata 1.6 has no return type and this PR add one.

That argument seems invalid: it *is* legal to add a return type when implementing/overriding (https://3v4l.org/rq2pd) (it isn't to remove, https://3v4l.org/oMqN5).

Would you accept a PR that changes `"^2.0"` to `"^1.6 || ^2.0"` (if the CI passes)?

Reply:
Of course! Thank you for noticing that

Reply:
Thanks, see #1212